README.md

```diff
@@ -1 +1,71 @@
-# hw-event-proxy-operator
+# Hardware Event proxy Operator
+## Table of Contents
+
+- [HW-Event-Proxy Operator](#hw-event-proxy operator)
```

Comment:
Need a dash here for the link to work: #hw-event-proxy-operator

Reply:
Fixed